### PR TITLE
feat: add mappers

### PR DESCRIPTION
**Pull request by dejwi**
### Task Description
[SCOUT-228](https://playmakerpro.atlassian.net/browse/SCOUT-228?atlOrigin=eyJpIjoiY2I4MThiZDQ2OGY4NDUwMDllYjljMDA0N2ZjNGMwMGQiLCJwIjoiaiJ9)

- fixed rating filtering on notes&reports

not fun types

### CODE REVIEW

File: src/pages/notes/index.tsx [hunk 1]
```diff
@@ -87,7 +88,11 @@ const NotesPage = () => {
     limit: rowsPerPage,
     sortBy: sortBy as NotesSortBy,
     sortingOrder: order,
-    ...mapFiltersStateToDto(filters),
+    ...mapFiltersStateToDto(
+      mapFilterFormDataToFiltersDto(
+        filters as any,
+      ) as unknown as NotesFiltersState,
```
Comment (jakubfiglak) [inline src/pages/notes/index.tsx:94]: What the hell is this 🤣 Please clean up these types, I'm sure there's a better way to handle this. Please remember that using `as` is a last resort - it basically makes Typescript useless

**Review by jakubfiglak — CHANGES_REQUESTED** [review]
Please clean up filter types:)

**Review by jakubfiglak — CHANGES_REQUESTED** [review]
Pretty close but there's a way to do it without using type casting at all:)

**Comment by dejwi** [comment]
😅

**Review by jakubfiglak — APPROVED** [review]
🎉

**Pull request merged**